src/AppBundle/Entity/SocialEvent.php

```diff
@@ -0,0 +1,236 @@
+<?php
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="event")
+ * @ORM\Entity(repositoryClass="AppBundle\Entity\Repository\SocialEventItemRepository")
+ */
+class SocialEvent
+{
+
+    /**
+     * @var Department
+     *
+     * @ORM\ManyToOne(targetEntity="Department")
+     * @@ORM\JoinColumn(referencedColumnName="id")
+     */
+    private $department;
+
+
+    /**
+     * @var Semester
+     *
+     * @ORM\ManyToOne(targetEntity="Semester")
+     * @ORM\JoinColumn(referencedColumnName="id")
+     */
+    private $semester;
+
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string")
+     * @Assert\NotBlank
+     */
+    private $title;
+
+    /**
+     * @ORM\Column(type="string")
+     */
+    private $description;
+
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $startTime;
+
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $endTime;
+
+    /**
+     * @var Role
+     * @ORM\ManyToOne(targetEntity="Role")
+     * @ORM\JoinColumn(referencedColumnName="id")
+     *
+     */
+    private $role;
+
+    /**
+     * Constructor.
+     */
+    public function __construct()
+    {
+    }
+
+    ## TODO: Burde det være en location?
+
+    ## TODO: Legg til i kalender?
+
+    ## TODO: Stand event?
+
+    ## TODO: E-mail event.
+
+    ## TODO : ICAL SUPPORT.
+
+    ## TODO : LEGG TIL HVEM SOM SKAL INVITERES
+
+    ## TODO : LISTE FOR PÅMELDE
+
+    ## TODO: BILDE
+
+    ## TODO: OVERSIKT OVER PÅMELDTE OG IKKE PÅMELDTE
+    /**
```

Comment:
Vi trenger ikke TODO kommentarer i koden for fremtidige feature requests. Det er bedre å lage issues på github for dette, så kan vi diskutere rundt det.